posthog/api/test/test_user.py

```diff
@@ -1,13 +1,428 @@
+import uuid
 from unittest.mock import patch
 
 from rest_framework import status
 
 from posthog.models import Team, User
-from posthog.models.organization import OrganizationMembership
+from posthog.models.organization import Organization, OrganizationMembership
 from posthog.test.base import APIBaseTest
 
 
-class TestUser(APIBaseTest):
+class TestUserAPI(APIBaseTest):
```

Comment:
Could move this in separate file to keep files smaller

Reply:
Well I actually think this is the class that makes sense keeping here. The other one is just a legacy class to be kept around until we remove the old endpoint.